test/rws_rapid_test.h

```diff
@@ -0,0 +1,26 @@
+#include <abb_librws/rws_rapid.h>
+
+namespace abb :: rws
+{
+    struct MockRAPIDAtomic
+    :   public RAPIDAtomicTemplate<std::string>
+    {
+        MockRAPIDAtomic(const std::string& value = "") 
+        :   RAPIDAtomicTemplate(value) {}
+
+        std::string getType() const override
+        {
+            return "mock_string";
+        }
+        
+        void parseString(const std::string& value_string) override
+        {
+            value = value_string;
+        }
+
+        std::string constructString() const override
+        {
+            return "" + value;
+        }
+    };
```

Comment:
What is the purpose of this type? Why not use `RAPIDString`?

Reply:
I thought about it but that way the UTests are safe from any change in the behavior of RAPIDString. For me the UTests for RAPIDArrays should break because of a change in the behavior of RAPIDString so that is why I created this type.
But maybe I overthought this

Reply:
This is a very thorough approach :-)